python/ray/worker.py

```diff
@@ -1998,6 +2002,8 @@ def connect(info,
     # the correct driver.
     if mode != WORKER_MODE:
         worker.task_driver_id = ray.ObjectID(worker.worker_id)
```

Comment:
I'm pretty sure this "driver ID" is different from the task ID of the driver task.

Reply:
Oops, I mixed this up with the `worker.current_task_id`. Thanks!